@xen-orchestra/web-core/lib/components/info/UiInfo.vue

```diff
@@ -0,0 +1,56 @@
+<!-- v2 -->
+<template>
+  <div class="vts-ui-info">
+    <UiIcon class="icon" :color :icon />
+    <p class="message"><slot /></p>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import type { IconColor } from '@core/types/color.type'
+import {
+  faCheckCircle,
+  faExclamationCircle,
+  faInfoCircle,
+  faXmarkCircle,
+  type IconDefinition,
+} from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+type Props = {
+  status: 'info' | 'success' | 'warning' | 'danger'
+}
+
+const props = defineProps<Props>()
+
+defineSlots<{
+  default(): any
+}>()
+
+const states: Record<Props['status'], { icon: IconDefinition; color: IconColor }> = {
+  info: { icon: faInfoCircle, color: 'normal' },
+  success: { icon: faCheckCircle, color: 'success' },
+  warning: { icon: faExclamationCircle, color: 'warning' },
+  danger: { icon: faXmarkCircle, color: 'danger' },
+}
+
+const icon = computed(() => states[props.status].icon)
+const color = computed(() => states[props.status].color)
+</script>
+
+<style lang="postcss" scoped>
+.vts-ui-info {
+  align-items: center;
```

Comment:
To handle the case when the text message is long and spans on multiple lines, I think it would be better to use `align-items: start;`. When the text in on a single line, there is not much visual difference between `center` and `start` alignments. What do you think?
```suggestion
  align-items: start;
```